docs/guide/api-environment-frameworks.md

```diff
@@ -75,7 +77,7 @@ app.use('*', async (req, res, next) => {
   // 3. Load the server entry. import(url) automatically transforms
   //    ESM source code to be usable in Node.js! There is no bundling
   //    required, and provides full HMR support.
-  const { render } = await environment.runner.import('/src/entry-server.js')
+  const { render } = await environment.runner.import('/src/serverEntry.js')
```

Comment:
Is there a reason for the name change here? In my SSR templates I usually use `entry-server` instead

Reply:
I needed one less character 😬 
Reverted, I don't know how to make this one work